### PR TITLE
Api integration

### PR DESCRIPTION
**Pull request by anandsr21harshit**
- Product Listing page will now fetch data from backend data base using axios
- Some changes in Home Page UI
- Hamburger icon is now made active

### CODE REVIEW

File: src/pages/Login.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function Login() {
                 <input className="input input-primary" type="text" placeholder="Type Username" id="userName"/>
                 <label for="Password">Password*</label>
                 <input className="input input-primary" type="password" placeholder="Type Password" id="password"/>
-                <a href="">Forgot password?</a>  
+                <a href='../'>Forgot password?</a>  
             </div>
             <button className="btn btn-secondary">Login</button>
             <p className="no-account">Don't have an account? <Link to="/signup">Sign Up</Link></p>
```
Comment (OmKakatkar) [inline src/pages/Login.js:18]: use `htmlFor` instead of `for`
use `Link` instead of `a` tag
Can be done in a future PR

**Review by OmKakatkar — APPROVED** [review]
Like:
* Meaningful variable names
* Separation of Pages and Components

Suggestion:
* Add a _redirects file to public folder to enable client side routing in Netlify

File: src/components/Toast.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import "../css/toast.css"
+
+function Toast() {
+  return (
+    <div className='toast-container'>
+        <h3>Added to Cart</h3>
+    </div>
+  )
```
Comment (shrey27) [inline src/components/Toast.js:9]: Its good you are using your own toast component

**Review by shrey27 — APPROVED** [review]
overall code looks good
PR is ready to merge

File: src/pages/Login.js (hunk 1 above)
Comment (anandsr21harshit) [reply src/pages/Login.js:18]: Will keep that in mind

**Pull request merged**